orchestration/dagster_orchestration/hca_utils/main.py

```diff
@@ -13,6 +17,20 @@ def error(self, message):
         sys.exit(2)
 
 
+def get_api_client(host: str) -> RepositoryApi:
+    # get token for jade, assumes application default credentials work for specified environment
+    credentials, _ = google.auth.default()
+    credentials.refresh(Request())
+
+    # create API client
+    config = Configuration(host=host)
+    config.access_token = credentials.token
+    client = ApiClient(configuration=config)
+    client.client_side_validation = False
```

Comment:
can these property assignments be done in the constructors instead?

Reply:
Surprisingly it doesn't look like it